src/keyboard/textinput.js

```diff
@@ -61,9 +61,21 @@ var TextInput = function(parentNode, host) {
         }
         if (options.role) {
             text.setAttribute("role", options.role);
-        }
+        }     
+    };
+    this.setAriaLabel = function() {
```

Comment:
can we maybe merge two functions here: `setAriaOptions` and `setAriaLabel`? They seem serving similar purposes.

Reply:
If we merge the functions, we'd need to make sure that whenever we call `setAriaLabel` we pass the same `options` as the last time we called `setAriaOptions` which I think would be a hassle given that e.g. autocomplete extensions also call this function.